lib/vfscore/file.c

```diff
@@ -52,6 +53,8 @@ int fdrop(struct vfscore_file *fp)
 		UK_CRASH("Unbalanced fhold/fdrop");
 
 	if (prev == 1) {
+		eventpoll_notify_close(fp);
+
 		/*
```

Comment:
Is there an indent issue here or is Github bugging out for me?

Reply:
Looks good on my side, also on github
![image](https://user-images.githubusercontent.com/8928900/179791357-8ccfcf52-821f-49fb-9280-0dc5ec06030d.png)